GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift

```diff
@@ -28,8 +30,13 @@ final class HarvestWidget: WidgetControlling {
         let emptyWidgetViewModel = HarvestWidgetViewModel(lastDayChartModel: emptyCircleChartModel,
                                                           lastNDaysChartModel: emptyCircleChartModel,
                                                           numberOfLastDays: numberOfDays)
+        widgetView = HarvestWidgetView.fromNib()
         widgetView.configureWithViewModel(emptyWidgetViewModel)
 
+        let viewModel = WidgetTemplateViewModel(title: "HARVEST", subtitle: "USERS BILLED HOURS REPORT", contentView: widgetView)
```

Comment:
I always write in a manner of `"Harvest".localized.uppercaseString` (just in case there will be a localized version), and it was pointed to me in another PRs.